sympy/functions/elementary/hyperbolic.py

```diff
@@ -209,6 +209,11 @@ class cosh(HyperbolicFunction):
     sinh, tanh, acosh
     """
 
+    @property
+    def is_positive(self):
+        if self._eval_is_real():
```

Comment:
@jksuom, this is the definition of the _eval_is_real() function-

def _eval_is_real(self):
        return self.args[0].is_real

So basically it checks whether the parameter passed (in this case, x) is real, and not whether the function cosh is real valued.